### PR TITLE
CCR-563: add table error / no data screen

### PR DESCRIPTION
**Pull request by suchak1**
### Proposed Changes
_Please use the Jira Key followed by the changes_

- JIRAKEY-1234
  - Some change
  - Another change

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [ ] I have verified that my changes have not introduced new lint errors
- [ ] I have added a GitHub labels to the PR for either _patch_, `minor`, or **major**

### CODE REVIEW

File: lib/components/table/error.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import PropTypes from 'prop-types';
+import styles from './error.css';
+import AttentionIcon from '../icon/AttentionIcon';
+
+const VARIANT = {
+  ERROR: 'ERROR',
+  NO_DATA: 'NO_DATA',
+};
+
+const TableError = (props) => {
```
Comment (ntrevino-virtru) [inline lib/components/table/error.js:15]: I'd like to see a test for this component. Not a big deal though. Feel free to add a story for it.

File: lib/components/table/error.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        Encrypt or decrypt something and{' '}
+        <span data-testid="audit-try-again-btn" onClick={onClick} className={styles.underline}>
+          try again
+        </span>
+        .
+      </span>
+    </>
+  );
+
+  if (variant === VARIANT.ERROR) {
```
Comment (ntrevino-virtru) [inline lib/components/table/error.js:32]: This feels like the content sections should be pulled out into separate components and those components would pass the contents as `children` to the actual display component. So `TableError` would take the variant, delegate to the variant component which would pass `children` to `TableErrorDisplay`.

Probably not necessary yet though, so feel free to ignore.

**Review by ntrevino-virtru — APPROVED** [review]

File: lib/components/table/error.js (hunk 1 above)
Comment (suchak1) [reply lib/components/table/error.js:15]: For this repo, I believe we only add stories and then do visual regression testing. And then we test functionality inside the client application.

Comment (ntrevino-virtru) [reply lib/components/table/error.js:15]: Gotcha. Not sure I'm a fan of that, but if that's the convention, then :+1:

**Review by dillonthompson — APPROVED** [review]

**Pull request merged**